runtime/common/src/apis.rs

```diff
@@ -16,7 +16,14 @@
 
 #[macro_export]
 macro_rules! impl_runtime_apis_plus_common {
-	{$($custom:tt)*} => {
+	{
+        benchmark_type_aliases: {
+            $($bench_type_aliases:tt)*
+        }
+        custom_impls: {
+            $($custom:tt)*
+        }
+    } => {
```

Comment:
If we no longer need this, lets revert it. It will conflict with the changes I will add for the bridge mvp.